src/Blockcore.Indexer.Angor/Storage/Mongo/AngorMongoData.cs

```diff
@@ -66,16 +67,149 @@ public class AngorMongoData : MongoData, IAngorStorage
             .Where(_ => _.AngorKey == projectId)
             .Sum(s => s.AmountSats);
 
+         var spendingSummery = await GetSpentProjectFundsSplitToFounderAndPenalty(projectId);
+
          return new ProjectStats
          {
             InvestorCount = total,
             AmountInvested = sum,
+            AmountSpentSoFarByFounder = spendingSummery.founderSpent,
+            AmountInPenalties = spendingSummery.investorWithdrawn,
+            CountInPenalties = spendingSummery.invetorTrxCount
          };
       }
 
       return null;
    }
 
+
+   private async Task<(long founderSpent, long investorWithdrawn, int invetorTrxCount)> GetSpentProjectFundsSplitToFounderAndPenalty(string projectId)
+   {
+      var founderSummery = "founder"; var investorSummery = "investor"; var total = "total"; var trxCount = "trxCount";
+
+      var outputsSpentSummery = await mongoDb.InvestmentTable.Aggregate(PipelineDefinition<Investment, BsonDocument>.Create(
+            new BsonDocument("$match",
+               new BsonDocument("AngorKey", projectId)),
+            new BsonDocument("$lookup",
+               new BsonDocument
+               {
+                  { "from", "Input" },
+                  { "localField", "TransactionId" },
+                  { "foreignField", "Outpoint.TransactionId" },
+                  { "as", "inputs" }
+               }),
+            new BsonDocument("$unwind", "$inputs"),
+            new BsonDocument("$match",
+               new BsonDocument("inputs.Address", "none")), //Remove change address
+            new BsonDocument("$group",
+               new BsonDocument
+               {
+                  { "_id",
+                     new BsonDocument
+                     {
+                        { "TransactionId", "$TransactionId" },
+                        { "TrxHash", "$inputs.TrxHash" }
+                     } },
+                  { "spent",
+                     new BsonDocument("$sum", "$inputs.Value") },
+                  { "numTrx",
+                     new BsonDocument("$sum", 1) }
+               }),
+            new BsonDocument("$group",
+               new BsonDocument
+               {
+                  { "_id",
+                     new BsonDocument("$cond",
+                        new BsonDocument
+                        {
+                           { "if",
```

Comment:
What? conditions in the pipeline? this is insane. doing such a query in sql would be far easier.